.github/CONTRIBUTING.md

```diff
@@ -31,70 +31,93 @@ If you at any point have any questions please read: [I have a question related t
+
+<!-- General description of the contribution model -->
+In broad strokes the process is: submit an issue, get it refined, implement it, get it reviewed. Once reviewed it will automatically be merged to master. 
+
+Actionable checklists for these steps can be found further down in this section. The following highlight some of these steps to explain what they are and why they exist: 
+
+<!-- Why we do UX refinement?-->
+**UX Refinement**: If an issue involves visual changes then UX refinement must take place. This consists of UXers planning how the issue can be solved in a way that aligns with the design system. The `NOT UX refined`-label will be removed from the issue by team Kirby, once it has been done. If a pull-request with visual changes is submitted for an issue that has not been UX refined; there is a good chance that the UX review process may result in a bunch of changes. 
+
+<!--- Why do we do Tech refinement? -->
+**Tech Refinement**: This should happen before any code is written for an issue. Here it is discussed and agreed on how an issue should be implemented the best. How long it takes depends on the scope of the issue. For small issues it might be nothing more than an agreement via a direct message. Other times it is a full-blown meeting with multiple participants. The `NOT Tech refined`-label will be removed from the issue by team Kirby once it has been done. If a pull-request is submitted for an issue that has not been tech refined; changes and problems that could have been tackled from the beginning, might arise from the code-review. 
+
+**Publish a WIP draft**: We encourage contributors to publish a draft pull-request with a WIP implementation of their issue to get feedback. This can advantageously be done before tests are written. By doing this misunderstandings and issues can be tackled early on; here they are often easier to correct. If this is not done, they might not be discovered until the code-review. Tests might have to be changed. So therefore we encourage contributors to use the opportunity to get some feedback. 
+
+<!-- Why do a self-review? -->
+**Self-review**: This consists of code-reviewing your own pull-request, following guidelines found in: [the good self review](#self-review). This is your chance to catch any low-hanging fruits and common issues before the reviewer does. If this is not done, an unneccessar amount of minor changes might be required during code review. This wastes both yours and the reviewers time. 
+
+<!-- Why we do UX reviews? -->
+**UX review**: If the implementation of the issue resulted in visual changes - then it has to be UX reviewed. This is to make sure that the introduced changes align with the design system. We strive to have every issue that introduces visual changes UX reviewed. If this is not done and the changes are merged to master, we might recieve issues about components not being properly aligned with the design system anymore from the users of Kirby. 
```

Comment:
Perhaps we should use a [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) for these. They're rarely used but this is a perfect use case 😊

Reply:
That's pretty neat! - I'm gonna throw those in :)

Reply:
Should be solved by new commits ☀️